src/FilterBundle/DependencyInjection/BestItCommercetoolsFilterExtension.php

```diff
@@ -32,6 +32,9 @@ public function load(array $configs, ContainerBuilder $container)
         $container->setAlias('best_it_commercetools_filter.request.client', $config['client_id']);
         $container->setAlias('best_it_commercetools_filter.generator.url', $config['url_generator_id']);
 
+        // Set parameters
+        $container->setParameter('best_it_commercetools_filter.cache_life_time', $config['cache_life_time']);
```

Comment:
It's easy to create a new major tag (5.x) - but I'd like to avoid it. If we now accept a Breaking Change for every "small adjustment" the updates will be more difficult in the future.

My wish would be if we can carry out updates in all projects very quickly and simply without having to carry out a "migration".

Better then collect points and publish a major update - and not for any small PR.

Discuss and decide both of you. I do not want to interfere from my holiday too much. ;-)